loading.bs

```diff
@@ -381,6 +391,38 @@ for="certificate">extensions</dfn> map ([=Certificate Extensions=]) from OIDs to
 A <dfn>certificate chain</dfn> is a [=list=] of [=augmented certificates=], of
 which the first [=list/item=] is the <dfn for="certificate chain">leaf</dfn>.
 
+<h3 dfn-type=dfn export>Signed Exchange report</h3>
+
+A signed exchange report is a [=struct=] with the following items:
+
+<dl dfn-for="signed exchange report">
+: <dfn>result</dfn>
+:: The result string of loading signed exchange. This must be one of an empty
```

Comment:
I'm uncomfortable that it's allowed to be an empty string. https://pr-preview.s3.amazonaws.com/horo-t/webpackage/pull/374.html#ref-for-signed-exchange-report-result%E2%91%A0, especially, results in a type of just "sxg." in that case. I think you're just using that as a default, so maybe:

```suggestion
:: The result string of loading signed exchange. This must be unset or one of "<dfn>`ok`</dfn>", ...
```